firmware/common/lcd_ili9341.cpp

```diff
@@ -416,6 +421,208 @@ void ILI9341::drawBMP(const ui::Point p, const uint8_t * bitmap, const bool tran
 	}
 }
 
+/*
+	Read a RAW image from SD card. (Mainly for splash screen)
+	RAW image structure:
+		First 3 bytes: 	file header "RAW"
+		Byte 4:			file type, 0: RGB565, 1: RGB, 2: RGBA (alpha ignored)
+		Byte 5-6:		height
+		Byte 7-8:		width
+*/
+bool ILI9341::drawRAW(const ui::Point p, const std::string file) {
```

Comment:
Why is the original drawBMP code replicated 3 times now? This needs to be cleaned up

Reply:
In fact these codes are different from drawBMP.
drawBMP only loads RLE coded 16-bit BMP from RAM.
Maybe drawRAW can be cleaned up (so do that Python script), as now normal BMP file can be loaded.